docs/blog/20230401-Daeraxa-AprUpdate.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Community Update
+author: Daeraxa
+date: 2023-04-01
+category:
+  - news
+  - log
+tag:
+  - update
+---
+
+Another dose of our regular monthly community update!
+
+<!-- more -->
+
+# Welcome to the April Community Update (don't worry, no fooling going on here)!
+
+Hi everyone and welcome to the April instalment of our monthly update that keeps keeps you informed about what is going on in the background of Pulsar, work in progress, community contributions and more!
```

Comment:
```suggestion
Hi everyone and welcome to the April installment of our monthly update that keeps keeps you informed about what is going on in the background of Pulsar, work in progress, community contributions and more!
```

Reply:
@Spiker985 mentioned this one above which has been committed. (Basically it seems its UK vs US spelling but as its meant to be from the team then it goes to US spelling, didn't know there was even a difference...)

Reply:
I've made some changes to this section as well to reflect that we already added a "good" badge along with an example.

Reply:
I appreciate all of that, I've applied my final review here, so I'll close this one out